spec/datadog/tracing/contrib/grpc/support/grpc_helper.rb

```diff
@@ -1,12 +1,21 @@
 require 'grpc'
-
 require 'spec/support/thread_helpers'
 
+if RUBY_VERSION < '2.3'
+  require_relative './gen/grpc-1.19.0/test_service_services_pb'
+else
+  require_relative './gen/test_service_services_pb'
+end
```

Comment:
Can this be replaced by requiring [this file?](https://github.com/DataDog/dd-trace-rb/pull/2620/files#diff-f7651db66dc15d9c30b482d82173a8403282611f14e041cb61f9359e8da28e5d)

Reply:
I think so, thanks for catching this up!